targets/pna_nic/.gitignore

```diff
@@ -0,0 +1,4 @@
+/pna_nic
+pnic_runtime
+pna_nic_CLI
+externs
```

Comment:
I feel like this doesn't need to be here?

Reply:
The first three lines represent the binaries created during the build process.

Regarding the fourth line, I have an `externs` directory in my local repo that I plan to add in later commits. For convenience, I added externs to the `.gitignore` file to temporarily exclude it in this commit.

Will remove fourth line.